db/schema.rb

```diff
@@ -37,8 +37,8 @@
     t.integer "points_for_win", default: 3
     t.integer "points_for_draw", default: 1
     t.integer "points_for_lose", default: 0
-    t.boolean "has_place_3_match", default: true
     t.text "image_data"
+    t.boolean "has_place_3_match", default: true
```

Comment:
You could discard/revert this change as was well the one from line 154/155. I don’t know why this always happens—the same re-ordering of existing columns in the Schema happens on my machine as well when running newly created migrations.

Reply:
@bakoe Is there a quick and easy way to do what you recommend? Otherwise I would suggest to just let rails be weird in this case... 🤪

Reply:
@lheilmann I can think of nothing quicker than manually adding a commit that changes `schema.rb` to be exactly the same as the one on `dev` with only the renamed field you introduced as difference. This would lead to a diff of this PR that would only contain you re-naming but not the apparent re-ordering of existing database columns.

However, I think it's pretty irrelevant – furthermore, we can't know for sure that the next person running another migration would not run into these re-ordering “issues” again. Thus, I would suggest just leaving the `schema.rb` changes in your PR as they are at the moment.

Reply:
Maybe it's an anti-pattern to manually change the `schema.rb`, so leaving it as it is now (created by `db:migrate`) is probably the best practice.

Reply:
Alright!